app/lib/webfinger.rb

```diff
@@ -6,6 +6,7 @@ class GoneError < Error; end
   class RedirectError < Error; end
 
   class Response
+    ACIVITYPUB_READY_TYPE = ['application/activity+json', 'application/ld+json; profile="https://www.w3.org/ns/activitystreams"'].freeze
```

Comment:
I think this should be `ACTIVITYPUB_READY_TYPE` (a `T` is missing).

And while I am nitpicking :slightly_smiling_face: : You might want to add a newline to separate the constant from the `attr_reader` below.